src/Stratis.Bitcoin/Connection/ConnectionManager.cs

```diff
@@ -148,6 +152,22 @@ public IReadOnlyNetworkPeerCollection ConnectedNodes
 
             this.Servers = new List<NetworkPeerServer>();
             this.connectedNodes = new NetworkPeerCollection();
+
+            // Don't start the peer discovery f we have specified any nodes using the -connect arg.
```

Comment:
This comment suggests that the next piece of code deals with having the `-connect` specified, but really it's with the `-connect` not specified.

Reply:
Also, there is a missing `i` at `if`.

Reply:
Hey mate, I think you reviewed a earlier commit here?